pydm/widgets/spinbox.py

```diff
@@ -38,38 +42,31 @@ def event(self, event):
         ----------
         event : QEvent
         """
-        if (event.type() == QEvent.KeyPress):
-            ctrl_hold = self.app.queryKeyboardModifiers() == Qt.ControlModifier
+        if event.type() == QEvent.KeyPress and \
+           self.app.queryKeyboardModifiers() == Qt.ControlModifier:
 
-            if ctrl_hold and (event.key() == Qt.Key_Left):
+            if event.key() == Qt.Key_Left:
                 self.step_exponent = self.step_exponent + 1
                 self.update_step_size()
                 return True
-
-            if ctrl_hold and (event.key() == Qt.Key_Right):
-                self.step_exponent = self.step_exponent - 1
-
-                if self.step_exponent < -self.decimals():
-                    self.step_exponent = -self.decimals()
-
+            elif event.key() == Qt.Key_Right:
+                self.step_exponent = max(self.step_exponent - 1,
+                                         -self.decimals())
                 self.update_step_size()
                 return True
 
-            if (event.key() == Qt.Key_Up):
-                self.setValue(self.value + self.singleStep())
-                self.send_value()
```

Comment:
The way it is implemented the value will be sent as soon as the user press the UP Key. However when the user clicks on the widget arrows the value will only be sent when Return is pressed. Should I remove the send_value in this line?